packages/keychain/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@blockstack/keychain",
-  "version": "0.12.2",
+  "version": "0.13.0-beta",
```

Comment:
Can you revert this change? Otherwise it'll mess up the auto-publish setup. And, for local development, try out yalc. It is pretty nice once you figure out the commands.

Reply:
Just tried it.

Does yalc add commits? Not sure where the version named commit came from.